File: Python/ex3.py
# 3. Sort numbers in an array

def sort(list):
    length = len(list)
    end = False
    for i in range (length):
        for j in range (0, length - 1):
            if list[j] > list[j+1]:
                list[j], list[j+1] = list[j+1], list[j]
                end = True
        if end == False: break
    return list

print(sort([1,5,2,6,3,4,127]))

File: JavaScript/ex3.js
// 3. Sort numbers in an array

const prompt = require ("prompt-sync")();

function sort(list){
    let end = false;
    for (i = 1; i <= list.length; i++){
        for (j = 0; j <= list.length - 1; j++){
            if (list[j] > list[j + 1]){
                let temp = list[j]
                list[j] = list[j + 1] 
                list[j + 1] = temp
                end = true;
                console.log(list)
            }
        }
        if (end === false){
            break;
        }
    }
    return list;
}
console.log(sort([1, 5, 3, 6, 2, 9, 7]))